packages/cactus-plugin-ledger-connector-iroha2/src/main/typescript/cactus-iroha-sdk-wrapper/client.ts

```diff
@@ -820,6 +822,9 @@ export class CactusIrohaV2Client {
       this.prerequisitesProvider.getApiHttpProperties(),
       signedTx,
     );
+    //counter will increase here 
```

Comment:
This is small nitpick, but could you put space between `//` and the comment? This is a common practice in the sources

Reply:
Plus one. I think this is also one of the reasons why the `yarn lint` CI check is failing - the auto-formatter would add the space and then the CI would notice the diff and fail the check.

Reply:
Done.